index.html

```diff
@@ -43,8 +43,6 @@
         <a href="#"><img src="mail.png" alt="E-Mail" class="navbar1" /></a>
       </nav>
     </header>
-
-    <!-- intropage -->
     <section class="mainSentences">
       <h1 class="mainSentenceText">
```

Comment:
✅ Good job on the implementation! However, I noticed a couple of areas that require adjustment to match the design more closely:

1️⃣ The height of the "mainSentenceText" element appears to be larger than what is specified in the design. To ensure consistency, please update your code to match the designated height in the design.

2️⃣ Additionally, the margin-top for this heading should be set to 120px, as indicated in the design. Currently, it is set to 52px. Please modify your code accordingly to align with the design specifications.

By making these adjustments, you will ensure that your project reflects the intended design and maintains visual consistency. Keep up the great work! 👍🎉